client/signup/steps/pressable-store/index.jsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import EmailValidator from 'email-validator';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Gridicon from 'components/gridicon';
+
+import LoggedOutForm from 'components/logged-out-form';
+import LoggedOutFormFooter from 'components/logged-out-form/footer';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import FormTextInput from 'components/forms/form-text-input';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+import HeroImage from './hero-image';
+
+export default React.createClass( {
+	displayName: 'PressableStoreStep',
+
+	propTypes: {
+		stepName: PropTypes.string.isRequired,
+		goToNextStep: PropTypes.func.isRequired,
+		signupDependencies: PropTypes.object.isRequired,
+	},
+
+	getInitialState() {
+		return {
+			email: '',
+			isValid: false,
+			error: null,
+		};
+	},
+
+	componentDidMount() {
+		if ( this._input ) {
+			this._input.focus();
+		}
+	},
+
+	onEmailChange( event ) {
+		const email = event.target.value;
+		const isValid = EmailValidator.validate( email );
+		const error = this.state.error && isValid
+			? null
+			: this.state.error;
+		this.setState( {
+			email,
+			isValid,
+			error,
+		} );
+	},
+
+	onSubmit( event ) {
+		event.preventDefault();
+
+		if ( ! this.state.isValid ) {
+			this.setState( {
+				error: this.translate( 'Please provide a valid email address.' ),
+			} );
+			return;
+		}
+
```

Comment:
We need to figure out a way to make sure the tracks event hits the server before navigating away.